src/scripting/lua_root.cc

```diff
@@ -360,10 +360,10 @@ void LuaEditor::__unpersist(lua_State* /* L */) {
  */
 
 /* RST
-World
------
+World Descriptions
+------------------
```

Comment:
```suggestion
World and Tribe Descriptions
----------------------------
```
This also needs references to tribe defining below – what used to be `LuaWorld` and `LuaTribes` is now fully merged into `LuaDescriptions`